### DISCUSSION TITLE
Change Duration Light is on for Ring Spotlight Camera

### DISCUSSION THREAD

**Discussion by latca** [dgreif/ring#701] [Q&A] [answered]
After turning on the battery powered Ring Spotlight Cam's light (both in the Ring App and Homekit), it will automatically turn off after 30 seconds.  In the homebridge-ring configuration setting the beamDurationSeconds doesn't seem affect the Spotlight Camera and I'm guessing that only works on the Ring Floodlight Camera.  

Is there a way through the API or scripting or some other means to increase the duration the light stays on?

**Comment by dgreif** [comment, marked as answer]
My guess is that the 30 seconds is a default imposed by Ring on battery cams to avoid draining their battery.  Unfortunately I don't have a spotlight cam to test this with, so I can't debug it and see if there is an option to keep it on longer.

**Reply by latca** [reply]
Thanks! For anyone looking into this please let me know of anything I can do to help with the research or to test something out, in a reasonably secure way.
